app/lib/apply_redis_connection.rb

```diff
@@ -0,0 +1,23 @@
+class ApplyRedisConnection
```

Comment:
Have you considered setting `Redis.current` in an initializer? If that works it would save some extra config, and new Redis-dependent dependencies (like Sidekiq) would automatically be able to fetch the connection.

Reply:
Good idea! Something like this? https://github.com/DFE-Digital/apply-for-teacher-training/pull/3818/commits/b1458eb28fddd28167bac822a2d538118aa9e293